precompiles/EcPairing.yul

```diff
@@ -448,17 +448,6 @@ object "EcPairing" {
                 ny0, ny1 := fp2Neg(y0, y1)
             }
 
-            /// @notice Check if a G2 point in jacobian coordinates is in the subgroup of the twisted curve.
-            /// @dev The coordinates are encoded in Montgomery form.
-            /// @param xp0, xp1 The x coordinate of the point.
-            /// @param yp0, yp1 The y coordinate of the point.
-            /// @param zp0, zp1 The z coordinate of the point.
-            /// @return ret True if the point is in the subgroup, false otherwise.
-            function g2IsInSubGroup(xp0, xp1, yp0, yp1, zp0, zp1) -> ret {
-                let xr0, xr1, yr0, yr1, zr0, zr1 := g2ScalarMul(xp0, xp1, yp0, yp1, zp0, zp1, TWISTED_SUBGROUP_ORDER())
-                ret := and(iszero(zr0), iszero(zr1))
-            }
-
```

Comment:
I am not clear about what was discussed internally, are we still moving forward with this change?